i18n/en.json

```diff
@@ -3070,7 +3070,10 @@
   "leave_private_channel_modal.leave": "Yes, leave channel",
   "leave_private_channel_modal.message": "Are you sure you wish to leave the private channel {channel}? You must be re-invited in order to re-join this channel in the future.",
   "leave_private_channel_modal.title": "Leave Private Channel {channel}",
-  "leave_team_modal.desc": "You will be removed from all public and private channels.  If the team is private you will not be able to rejoin the team.  Are you sure?",
+  "leave_team_modal_guest.desc": "**You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels}} and {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** You won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_private.desc": "**You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels} } on this team.** If the team is private you won't be able to rejoin it the team without an invitation from another team member. Are you sure?",
```

Comment:
```suggestion
  "leave_team_modal_private.desc": "**You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels} } on this team.** If the team is private you won't be able to rejoin it without an invitation from another team member. Are you sure?",
```

Reply:
@justinegeffen it is showing wrong from the files section, I have opened the actual file it is correct there, can you recheck this by opening this https://github.com/mattermost/mattermost-webapp/blob/2e2502234d7b1e7340de141424d5670e2c865f10/i18n/en.json#L3073?